doc/source/quickstart.rst

```diff
@@ -14,30 +14,27 @@ In the Python interpreter, you may then call::
 which should run without error if the package has been correctly installed.
 
 
-Apply 2D scattering to a 32x32 random image
+Apply 2D scattering to a 32×32 random image
 -------------------------------------------
 
-The following code imports ``torch`` and the ``Scattering2D`` class, which
-implements the 2D scattering transform. It then creates an instance of this
-class to compute the scattering transform at scale ``J = 2`` of a 32x32 image
+The following code imports ``numpy`` and the ``Scattering2D`` class, which implements the 2D scattering transform. It
+then creates an instance of this class to compute the scattering transform at scale ``J = 2`` of a 32×32 image
 consisting of Gaussian white noise::
 
-    import torch
+    import numpy as np
     from kymatio import Scattering2D
```

Comment:
`from kymatio.numpy import Scattering2D`

Reply:
Still says `from kymatio`.

Reply:
Never mind. Can't figure out why GH interface isn't showing the latest version here.